src/index.ts

```diff
@@ -1,34 +1,37 @@
 export = {
+  parser: '@typescript-eslint/parser',
   extends: 'eslint-config-standard',
   plugins: ['@typescript-eslint'],
   overrides: [
     {
       files: ['*.ts', '*.tsx'],
       rules: {
-        'camelcase': 'off', // in favor of TypeScript rule
-        'indent': 'off',
+        camelcase: 'off', // in favor of TypeScript rule
+        indent: 'off',
```

Comment:
I think that the new rule in Standard 14 is that all lines should have `'` if at least one row needs it 😄 🤔

Reply:
@LinusU Actually, we went with the rule that we only need `'` on lines where it's needed. In other words, this is okay:

```js
const obj = {
  'hi-there': 1,
  hey: 2
}
```